packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -137,6 +137,15 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   const typingIndicatorStyles = Object.assign({}, styles?.typingIndicator);
   const sendBoxStyles = Object.assign({}, styles?.sendBox);
 
+  const fileUploadButtonOnChange = (files: FileList | null): void => {
+    if (!files) {
+      return;
+    }
+    const fileUploads = Array.from(files).map((file) => new FileUpload(file));
+    adapter.registerFileUploads && adapter.registerFileUploads(fileUploads);
```

Comment:
If I upload 2 files, they finish uploading, then click to upload a third -- does this get all three files? If so do the registerFileUploads handle the two that have already been uploaded or will the developer be expected to call updateFileUpload to set as complete?

Reply:
Great point. Let me think about that. I don't think the current designs factors in that use case.

Reply:
Made a slight change in the `registerFileUploads` method and now this scenario should work
![image](https://user-images.githubusercontent.com/9782747/152413744-7796a092-4a64-4954-b5c6-e9124b373a3b.png)